spec/components/courses/international_students_component_spec.rb

```diff
@@ -32,15 +37,20 @@
         funding_type: 'fee',
         provider: provider,
```

Comment:
you could even have the providers declared inline to reduce the lines of code in file but either way whichever you prefer

Reply:
@JonF1 I tried doing this, but it seemed less readable to me? Is there another way?

```ruby
      course = build(
        :course,
        funding_type: 'fee',
        provider: build(
          :provider,
          can_sponsor_student_visa: false,
          can_sponsor_skilled_worker_visa: false,
        )
      )
```

Reply:
`course = build(
        :course,
        funding_type: 'fee',
        provider: build( :provider, can_sponsor_student_visa: false, can_sponsor_skilled_worker_visa: false)
      )`
      
Following the style in the `search_result_component_spec.rb`, i guess it's personal preference

Reply:
@JonF1 nice idea, done in 90d611d1954515702c2656e4345b4249c5655064. I also realised that only 1 of the attributes was relevant in each case, so could remove the other one.